Source/ModelGroups/Mesh_PrimitiveMode.cs

```diff
@@ -1,488 +1,356 @@
-                }
+                    Properties = properties,
+                    GLTF = CreateGLTF(() => new Runtime.Scene()
+                    {
+                        Nodes = new List<Runtime.Node>
+                        {
+                            new Runtime.Node
+                            {
+                                Mesh = new Runtime.Mesh
+                                {
+                                    MeshPrimitives = new List<Runtime.MeshPrimitive>
+                                    {
+                                        meshPrimitive
+                                    }
+                                },
+                            },
+                        },
+                    }),
+                };
             }
 
-            List<Vector3> noIndicesPositionsLineStrip = new List<Vector3>()
-            {
-                new Vector3( 0.5f,-0.5f, 0.0f),
-                new Vector3( 0.5f, 0.5f, 0.0f),
-                new Vector3(-0.5f, 0.5f, 0.0f),
-                new Vector3(-0.5f,-0.5f, 0.0f),
-                new Vector3( 0.5f,-0.5f, 0.0f),
-            };
-            List<Vector3> noIndicesPositionsTrianglestrip = new List<Vector3>()
-            {
-                new Vector3( 0.5f,-0.5f, 0.0f),
-                new Vector3( 0.5f, 0.5f, 0.0f),
-                new Vector3(-0.5f,-0.5f, 0.0f),
-                new Vector3(-0.5f, 0.5f, 0.0f),
-            };
-            Runtime.GLTF defaultModel = Common.SinglePlane(); // Only used to get the default indices
-            List<int> defaultModelIndices = new List<int>(defaultModel.Scenes[0].Nodes[0].Mesh.MeshPrimitives[0].Indices);
-            List<int> linesIndices = new List<int>
-            {
-                0, 3, 3, 2, 2, 1, 1, 0,
-            };
-            List<int> lineloopFanIndices = new List<int>
-            {
-                0, 3, 2, 1,
-            };
-            List<int> linestripIndices = new List<int>
-            {
-                0, 3, 2, 1, 0,
-            };
-            List<int> trianglestripIndices = new List<int>
-            {
-                0, 3, 1, 2,
-            };
-            List<int> pointsIndices = new List<int>();
-            for (int x = 0; x < noIndicesPositionsPoints.Count; x++)
+            void SetModePoints(List<Property> properties, Runtime.MeshPrimitive meshPrimitive)
             {
-                pointsIndices.Add(x);
+                List<Vector3> pointsPositions = new List<Vector3>();
+                List<Vector3> cornerPoints = new List<Vector3>()
+                {
+                    new Vector3( 0.5f,-0.5f, 0.0f),
+                    new Vector3(-0.5f,-0.5f, 0.0f),
+                    new Vector3(-0.5f, 0.5f, 0.0f),
+                    new Vector3( 0.5f, 0.5f, 0.0f),
+                    new Vector3( 0.5f,-0.5f, 0.0f)
+                };
+                for (int corner = 0; corner < 4; corner++)
+                {
+                    for (int x = 256; x > 0; x--)
```

Comment:
why not just start with a float?

Reply:
@bghgary Because this value is also being used to refer by index to elements in `List<Vector3> cornerPoints'. See lines 68-69.

Reply:
Ah, I see. I've made the fix.